docs/elements/examples/sign-up.mdx

```diff
@@ -0,0 +1,41 @@
+---
+description: Examples of pre-built sign-up flows with Clerk Elements.
+---
+
+# Sign-up
+
+## Basic sign-up
+
+<div className="flex flex-col gap-10 border pt-10 rounded-xl bg-gray-25">
+
+```tsx
```

Comment:
```suggestion
```tsx filename="sign-up-example.tsx"
```

Reply:
I believe lack of filename causes issues with the current redesign ([relevant discussion](https://clerkinc.slack.com/archives/C06CA0JUZ8D/p1714417049984169))